core-bundle/src/Resources/contao/library/Contao/FeedItem.php

```diff
@@ -113,12 +117,29 @@ public function addEnclosure($strFile, $strUrl=null, $strMedia='enclosure')
 			$strUrl = Environment::get('base');
 		}
 
+		$fileUrl = $strUrl . System::urlEncode($strFile);
 		$objFile = new File($strFile);
 
+		$imageSize = StringUtil::deserialize($imageSize, true);
+
+		if ($imageSize && $objFile->isImage)
+		{
+			$image = System::getContainer()->get('contao.image.image_factory')->create($rootDir . '/' . $strFile, $imageSize);
+
+			if ($image instanceof DeferredImageInterface)
+			{
+				System::getContainer()->get('contao.image.resizer')->resizeDeferredImage($image);
+			}
+
+			$fileUrl = $strUrl . System::urlEncode($image->getUrl($rootDir));
+			$relativeFilePath = str_replace($rootDir . '/', '', $image->getPath());
```

Comment:
this should probably use `PathUtils::makeRelative()` (not sure if that's the exact name 🙈 )

Reply:
Yup, it's called `Path`: :slightly_smiling_face: 

```php
$relativeFilePath = Path::makeRelative($image->getPath(), $rootDir);
```